src/boot/efi/meson.build

```diff
@@ -136,6 +136,45 @@ if have_gnu_efi
         endif
 endif
 
+sbat_substs = configuration_data()
+sbat_substs.set('sbat_project', meson.project_name())
+sbat_substs.set('PROJECT_VERSION', substs.get('PROJECT_VERSION'))
+sbat_substs.set('PROJECT_URL', substs.get('PROJECT_URL'))
+sbat_substs.set('sbat_distro_generation', get_option('sbat_distro_generation'))
+sbatvars = [['sbat_distro', 'ID'],
+            ['sbat_distro_summary', 'NAME'],
+            ['sbat_distro_url', 'BUG_REPORT_URL']]
+foreach sbatvar : sbatvars
+        value = get_option(sbatvar[0])
+        if value == ''
+                value = run_command('sed', '-n', 's/^' + sbatvar[1] + '=//p', '/etc/os-release').stdout().strip()
```

Comment:
maybe:

sh -c '. /etc/os-release && echo $ID'

Reply:
I thought that one must never source `/etc/os-release` file =) and that one must parse it. But reading the man page, i see that it is intended to be "shell compatible" for variable assignment.